### ISSUE TITLE
Kuiper Table Installation

### ISSUE THREAD

**Issue by DanielHHowell** [circstat/pycircstat#63]
Having difficulty with the Kuiper table - receiving this error when trying to run the test: "[Errno 2] No such file or directory: '/home/daniel/anaconda3/lib/python3.7/site-packages/data/kuiper_table.npy'". Tried manually downloading from the repo and placing it in the directory it was looking for but received an axis error so not sure if that was the right thing to do

**Comment by fabiansinz** [comment]
I guess you made sure that the file was actually placed there? Did you check the file right?

**Comment by DanielHHowell** [comment]
Yes I tried placing the file there, and I believe the axis error is because N>>500 in the lookup table

**Comment by dominikstrb** [comment]
I got the same error, because my N was too large. Does anyone know a fix for this?

**Comment by DanielHHowell** [comment]
I can't remember if I was able to get it working, but looking back at the project it seems for one reason or another I switched to using astropy.stats.kuiper

**Comment by philippberens** [comment]
In this case, you can just use the last line, and P will be < than the pvalue in the last line.

**Comment by dominikstrb** [comment]
Thanks. I believe this is what is tried to be achieved [here](https://github.com/circstat/pycircstat/blob/53d2efdf54c394fd9ecff8d47eaae165c1458fb0/pycircstat/tests.py#L506), but it does not work in my case. If I add ```-1``` to that line, it works.

If you would like your library to be pip-installable without the need for users to download additional files, you could use this in your setup: https://python-packaging.readthedocs.io/en/latest/non-code-files.html.

**Comment by philippberens** [comment]
sorry, i really should get back to being better on taking care of this code... new years resolutions!

**Comment by A-ep93** [comment]
Is this (using the value of the last line of the table (N=501) for all larger sizes of arrays) a safe choice for both cases when the hypothesis gets rejected/accepted?
